carma-messenger-core/traffic_incident/src/traffic_incident_node.cpp

```diff
@@ -150,8 +150,7 @@ bool TrafficIncidentNode::stopTrafficBroadcastCallback(
 void TrafficIncidentNode::spin_callback(void)
 {
   if (
-    traffic_worker_.getDownTrack() > 0 && traffic_worker_.getUpTrack() > 0 &&
-    traffic_worker_.getMinGap() > 0 && traffic_worker_.getAdvisorySpeed() > 0) {
+    traffic_worker_.getDownTrack() > 0 || traffic_worker_.getUpTrack() > 0 || traffic_worker_.getAdvisorySpeed() > 0) {
```

Comment:
so this will potentially have a same issue as before due to the floating point issue. Lets compare with EPSILON:
```
EPSILON = 1e-6;
if (std::fabs(value) > EPSILON ) 
```